MapboxNavigation/NextBannerView.swift

```diff
@@ -53,7 +53,7 @@ open class NextBannerView: UIView {
     override open func prepareForInterfaceBuilder() {
         super.prepareForInterfaceBuilder()
         maneuverView.isEnd = true
-        instructionLabel.instruction = [VisualInstructionComponent(type: .destination, text: "Next step", imageURL: nil, maneuverType: .none, maneuverDirection: .none)]
+        instructionLabel.instruction = [VisualInstructionComponent(type: .text, text: "Next step", imageURL: nil, maneuverType: .none, maneuverDirection: .none, abbreviation: nil, abbreviationPriority: NSNotFound)]
```

Comment:
`NSLocalizedString` around `Next step`?

Reply:
This is a placeholder.

Reply:
This placeholder is never shown on the UI, I reckon. 

(@frederoni mentioned that localization will be done at server-side... so cool 👍)